overlord/ifacestate/ifacemgr.go

```diff
@@ -46,9 +46,6 @@ func Manager() (*InterfaceManager, error) {
 // Connect initiates a change connecting an interface.
 //
 func Connect(change *state.Change, plugSnap, plugName, slotSnap, slotName string) error {
-	change.State().Lock()
-	defer change.State().Unlock()
-
 	task := change.NewTask("connect", fmt.Sprintf("connect %s:%s to %s:%s",
```

Comment:
Feels like these should be properly capitalized, and will likely be translated at some point. They'll show up in the UI by themselves when looking inside/under a Change, which also has a proper sentence as a summary.